src/ferui-components/tree-view/interfaces.ts

```diff
@@ -91,3 +91,10 @@ export enum TreeViewColorTheme {
   GRAY = 'gray',
   WHITE = 'white'
 }
+
+/**
+ * Tree View auto select a node interface
+ */
+export interface TreeViewAutoSelector<T> {
+  autoSelectNode(nodesArray: Array<TreeNodeData<T>>): TreeNodeData<T>;
```

Comment:
Please rename the interface into `TreeViewAutoNodeSelector` instead.

Reply:
Renamed!